File: spring-redis/src/main/java/com/example/redis/_demo/Student.java
package com.example.redis._demo;

public class Student {

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getSex() {
        return sex;
    }

    public void setSex(String sex) {
        this.sex = sex;
    }

    private int id;

    public Student(int id, String name, String sex) {
        this.id = id;
        this.name = name;
        this.sex = sex;
    }

    private String name;
    private String sex;


}

File: basic-code-go/student/student.go
package main

// 定义一个学生类型结构体
type student struct {
	id    int    //学号
	name  string //姓名
	class string
}

// 构造函数
func newStudent(id int, name, class string) *student {
	return &student{
		id:    id,
		name:  name,
		class: class,
	}
}
